pkg/apis/core/types_shoot.go

```diff
@@ -863,8 +863,13 @@ type KubeletConfig struct {
 	MaxPods *int32
 	// PodPIDsLimit is the maximum number of process IDs per pod allowed by the kubelet.
 	PodPIDsLimit *int64
+	// TODO(shafeeqes): Remove this field in gardener v1.89
+
 	// ImagePullProgressDeadline describes the time limit under which if no pulling progress is made, the image pulling will be cancelled.
 	// Default: 1m
+	// Only relevant for docker CRI.
+	//
+	// Deprecated: This field is deprecated and will be removed in Gardener release v1.89.
```

Comment:
```suggestion
	// Deprecated: This field is deprecated and will be removed in Gardener release v1.90.
```

Same comment applies for the v1beta1 types.